Lib/selectors.py

```diff
@@ -357,7 +357,7 @@ def register(self, fileobj, events, data=None):
             poller_events |= self._EVENT_WRITE
         try:
             self._selector.register(key.fd, poller_events)
-        except Exception:
+        except:
```

Comment:
This is very uncommon code. If it is needed we have much more general problem. Every `except ... raise` should be rewritten in similar way. Maybe even every `finally` block. This is too cumbersome and may even need syntax support or changing the semantic.